### PR TITLE
APEXCORE-351 ignore Unstable/Evolving classes from semver

### PR DESCRIPTION
**Pull request by otterc**

### CODE REVIEW

**Comment by otterc** [comment]
@PramodSSImmaneni please review 

CI failed because of a iteration test failure for which is addressed by another pull request.

**Comment by vrozov** [comment]
this is already handled by #226

**Comment by otterc** [comment]
The annotations are incorrectly specified in that pull request. I had fixed that in Malhar.

**Comment by otterc** [comment]
This pull request correctly specifies annotations in the config.

File: pom.xml [hunk 1]
```diff
@@ -363,6 +363,10 @@
               <onlyBinaryIncompatible>false</onlyBinaryIncompatible>
               <includeSynthetic>false</includeSynthetic>
               <ignoreMissingClasses>false</ignoreMissingClasses>
+              <excludes>
```
Comment (pramodin) [inline pom.xml:366]: How about using includes instead of excludes?

Comment (pramodin) [inline pom.xml:366]: Ignore

**Comment by vrozov** [comment]
It is fixed now in the PR along with the version change.

**Pull request closed without merging**